guava/src/test/java/com/fasterxml/jackson/datatype/guava/TestRange.java

```diff
@@ -94,6 +107,10 @@ public void testWrappedSerialization() throws Exception
         testSerializationWrapped(MAPPER, RangeFactory.lessThan(10));
         testSerializationWrapped(MAPPER, RangeFactory.singleton(1));
     }
+
+    public void testSerializationToString() throws Exception{
+        assertEquals(MAPPER.writeValueAsString(new Stringified(RangeFactory.open(1, 10))), "{\"r\":\"[1..10]\"}");
```

Comment:
```suggestion
        assertEquals(MAPPER.writeValueAsString(new Stringified(RangeFactory.open(1, 10))), "{\"r\":\"(1..10)\"}");
```

Since we are using `open` range.

Reply:
So the test supposed to fail, but is passing?

Reply:
Yep. Which makes me think the getStringFormat method is not quite right 👀

Reply:
The bigger problem is that the PR author @saydar31 is not passing CLAs, 🙉